frontend/src/components/habits/habit_show.jsx

```diff
@@ -31,19 +32,23 @@ class HabitShow extends React.Component {
       <div className="habit-show-container">
         <div className="habit-show-grid">
           <div className="habit-show-body">
-            <h1>{habit.title}</h1>
-            <button className="habit-delete"
-                    onClick={() => openConfirmHabitDeleteModal(habitId)} >
-              delete habit
-            </button>
-            <button className="resources-index-new-button"
-                    onClick={() => openNewResourceModal(habitId)}>
-              New Resource
-            </button>
-            <button className="edit-link"
-                    onClick={() => openEditHabitModal(habitId) } >
-              Edit
-            </button>
+            <h1>{habit.title}</h1> 
+            {currentUser && currentUser.id === habit.user ? (
```

Comment:
this is completely optional, but I just wanted to comment that this specific logic can be done in mapStateToProps, but the end-result is probably exactly the same